House-of-glass/src/App.css

```diff
@@ -61,4 +61,85 @@
     grid-template-rows: 15rem auto 12.5rem;
     width: 90%;
   }
+}
+
+.products-container{
+
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-areas: 
+  "filter text "
+  "filter products"
+  "filter products";
+
+}
+
+.products-text-container{
+  grid-area: text;
+
+}
+
+/* Filter Pills */
+.pill{
+  border: 2px solid black;
+  width: fit-content;
+  padding: 0.2rem 0.5rem;
+  border-radius: 40px;
+  margin-bottom: 20px;
+}
```

Comment:
you might wanna change the depth of border color and make them take the entire width available?

Reply:
Hi Tahir,
Yes. They look better now.
Thank you for the suggestion.